### DISCUSSION TITLE
Query Log - Filter by specific DNS blocklist

### DISCUSSION THREAD

**Discussion by chatbuddy21** [AdguardTeam/AdGuardHome#3819] [Ideas]
Hi,

On the Query Log, is it possible to add as filter the individual Blocklist, so it would be easier to determine which specific list is blocking it.

Right now there are four options available. If you filter by 'Blocked', it's quite hard to scroll since there are too many.

**Comment by ainar-g** [comment]
There is no such feature currently, and it probably won't be practical until #2290 is solved.  You can create an issue about it, if there is no such issue already to make sure that that functionality is considered.

**Comment by chatbuddy21** [comment]
Understood, I will just monitor #2290 progress
